src/QueryDescription/index.js

```diff
@@ -165,6 +166,14 @@ export function between(left: number, right: number): Comparison {
   return { operator: 'between', right: { values } }
 }
 
+export function like(value: string): Comparison {
+  return { operator: 'like', right: { value } }
+}
+
+export function sanitizeLikeString(value: string): string {
```

Comment:
we need tests for this. In databaseTests, we should test the behavior of querying with a sanitizedLikeString (where the string is something nasty, containing %, _, and regex-unsafe expressions)